src/registers/mod.rs

```diff
@@ -2,3 +2,5 @@
 
 pub mod control;
 pub mod model_specific;
+pub mod msr;
```

Comment:
I would prefer if we only had a single module for model specific registers. How about creating constants of type `Msr` in the `model_specific` module (only for the most important registers)?